lib/phlex/view.rb

```diff
@@ -43,6 +44,16 @@ def call(buffer = +"", view_context: nil, parent: nil, &block)
 			buffer
 		end
 
+		def method_missing(name, *args, **kwargs, &block)
+			view_class = ComponentRegistry.instance.fetch(name, self.class)
+
+			# we can include the slow method as fallback / when eager loading is disabled, omitted while trying it out
+			view_class ||= (@_wrapper || self).instance_eval name.to_s.gsub("_", "::")
```

Comment:
This never actually worked for non eager-loaded views. 😞

Reply:
I had a conversation about this with Xavier Noria yesterday https://twitter.com/joeldrapper/status/1581765885165178880